conf/mccleary.config

```diff
@@ -0,0 +1,118 @@
+params {
+  config_profile_description = 'Yale YCRC Configurations'
+  config_profile_contact = 'Ryan A. Hagenson'
+}
+
+profiles {
+  mccleary {
```

Comment:
```suggestion
```

Reply:
Ah I see now, the profile is not defined here, it is defined within the `nfcore_custom.config` file, so you can removed those lines and the indentation